README.md

```diff
@@ -144,10 +144,19 @@ things that authors can do to help keep the conversation moving along.
 ## Using Labels
 
 The following labels may be applied to enhancements to help categorize them:
-
-- `priority/important-soon` indicates that the enhancement is related to a
-top level release priority. These will be highlighted in the
-[this-week](this-week/) newsletters.
+- `priority/important-soon` indicates that the enhancement pull request is related to a
+top level release priority. These will be highlighted in the [this-week](this-week/)
+- `priority/high`  indicates that relative priority of the enhancement as compared
+ to the other enhancement pull requests
```

Comment:
[This comment][1] suggested that individual teams would be setting semantics, so do we need "to the other enhancement pull requests belonging to the relevant team"?  Or something that points out that this is a relative position in a per-team (or per-approver?) queue, and not a relative position in a single repo-wide queue?

[1]: https://github.com/openshift/enhancements/pull/1129#discussion_r879429987